src/KeyValidators.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace League\ISO3166;
+
+trait KeyValidators
+{
+    /**
+     * Assert that input looks like an alpha2 key.
+     *
+     * @param string $alpha2
+     *
+     * @throws \DomainException if input is not a string.
+     * @throws \DomainException if input does not look like an alpha2 key.
+     *
+     * @return string
+     */
+    public function validateAlpha2($alpha2)
```

Comment:
@alcohol an alternative would be to just have it not return anything but only throw exceptions, methods like `guardAgainstInvalidAlhpa2`.